app/models/form/page.rb

```diff
@@ -49,7 +49,7 @@ def depends_on_met(case_log)
 
     depends_on.any? do |conditions_set|
       conditions_set.all? do |question, value|
-        value.nil? ? case_log[question] == value : !case_log[question].nil? && case_log[question] == value
+        value.nil? ? eval("case_log.#{question}") == value : !eval("case_log.#{question}").nil? && eval("case_log.#{question}") == value
```

Comment:
Not sure about this one, but couldn't think of another way of getting nested/org field

Reply:
Yeah, it didn't.
Well, the original way to access the field value would be:
`case_log[key]`
but if we want to access `case_log.managing_organisation.provider_type` that would translate to `case_log["managing_organisation.provider_type"]` maybe I'm missing an obvious way to make it work here 🤔

Reply:
Something along the lines of:

```
case_log_value = if question.include?(".")
  parts = question.split(".")
  case_log.public_send(parts.first.to_sym).public_send(parts.second.to_sym)
else
  case_log[question]
end

value.nil? ? case_log_value == value : !case_log_value.nil? && case_log_value == value
```

Reply:
Can probably refactor that nicer to make it accept any number of chained methods but should work as an approach?